azurerm/internal/services/databoxedge/databox_edge_orders_resource.go

```diff
@@ -0,0 +1,586 @@
+							Computed: true,
+						},
+
+						"additional_order_details": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+
+						"update_date_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"shipping_address": {
+				Type:     schema.TypeList,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"address_line1": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+
+						"city": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeCity,
+						},
+
+						"country": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeCountry,
+						},
+
+						"state": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeState,
+						},
+
+						"postal_code": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgePostalCode,
+						},
+
+						"address_line2": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+
+						"address_line3": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+					},
+				},
+			},
+
+			"delivery_tracking_info": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"carrier_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"serial_number": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"tracking_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"tracking_url": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"order_history": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"additional_order_details": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+
+						"comments": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"update_date_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"return_tracking_info": {
```

Comment:
should this be
```suggestion
			"return_shipment": {
```

Reply:
I went with `return_tracking` hopes that's ok... it made more sense to me... if you really want me to update this to use shipment I can do that too... 🙂